### ISSUE TITLE
Fail to use transpose

### ISSUE THREAD

**Issue by hamelsmu** [hamelsmu/Issue-Label-Bot-Examples#129]
System information

OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 16.04.3
TensorFlow installed from (source or binary): source
TensorFlow version (or github SHA if from source): 1.13.1
Provide the text output from tflite_convert
2019-04-12 16:14:22.935998: I tensorflow/lite/toco/graph_transformations/graph_transformations.cc:39] Before Removing unused ops: 13 operators, 22 arrays (0 quantized)
2019-04-12 16:14:22.936163: I tensorflow/lite/toco/graph_transformations/graph_transformations.cc:39] Before general graph transformations: 13 operators, 22 arrays (0 quantized)
2019-04-12 16:14:22.939819: I tensorflow/lite/toco/graph_transformations/graph_transformations.cc:39] After general graph transformations pass 1: 6 operators, 14 arrays (0 quantized)
2019-04-12 16:14:22.939890: I tensorflow/lite/toco/graph_transformations/graph_transformations.cc:39] After general graph transformations pass 2: 5 operators, 12 arrays (0 quantized)
2019-04-12 16:14:22.939945: I tensorflow/lite/toco/graph_transformations/graph_transformations.cc:39] After general graph transformations pass 3: 5 operators, 12 arrays (0 quantized)
2019-04-12 16:14:22.939972: F tensorflow/lite/toco/graph_transformations/propagate_fixed_sizes.cc:1605] Check failed: axis < input_shape.dimensions_count() (76044600 vs. 4)
Aborted (core dumped)

# Copy and paste here
Also, please include a link to a GraphDef or the model if possible.

Any other info / logs

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.
ipts = tf.placeholder(dtype=tf.int32, shape=(None, 50, 1000), name='x')
res = tf.layers.conv1d(ipts, 100, 1)
res = tf.transpose(res, perm=[0, 2, 1]) # Here cause the BUG
sess = tf.Session()
sess.run(tf.global_variables_initializer())
converter = tf.contrib.lite.TFLiteConverter.from_session(sess, [ipts], [res])
tflite_model = converter.convert()
open('converted_model.tflite', 'wb').write(tflite_model)

testing: https://github.com/tensorflow/tensorflow/issues/27781
